src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimSearchRepository.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.cmc.claimstore.repositories;
+
+import org.skife.jdbi.v2.sqlobject.Bind;
+import org.skife.jdbi.v2.sqlobject.SqlQuery;
+import org.skife.jdbi.v2.sqlobject.customizers.RegisterMapper;
+import org.skife.jdbi.v2.sqlobject.customizers.SingleValueResult;
+import uk.gov.hmcts.cmc.claimstore.repositories.mapping.ClaimMapper;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.util.List;
+import java.util.Optional;
+
+@RegisterMapper(ClaimMapper.class)
+@SuppressWarnings("squid:S1214") // Pointless to create class for string statement
+public interface ClaimSearchRepository {
```

Comment:
Rename it to `CaseDBI`. That will get infected to the instance of `CaseRepository` which uses DB as a data store.